Initial release of the Poring village instance (#7293)
rathena/rathena: npc/re/instances/PoringVillage.txt

	areamonster 'map_begi$, 79,113, 92,113, "--en--", "MD_MARIN",6, .@event$;

	monster 'map_begi$,101,66, "--en--","MD_MARIN",1, .@event$;
	monster 'map_begi$,57,113, "--en--","MD_MARIN",1, .@event$;
	monster 'map_begi$,68,113, "--en--","MD_MARIN",1, .@event$;
	monster 'map_begi$,176,66, "--en--","MD_PORING",1, .@event$;
	monster 'map_begi$,158,68, "--en--","MD_PORING",1, .@event$;
	monster 'map_begi$,157,67, "--en--","MD_PORING",1, .@event$;
	monster 'map_begi$,149,66, "--en--","MD_PORING",1, .@event$;
	monster 'map_begi$,125,67, "--en--","MD_PORING",1, .@event$;
	monster 'map_begi$,121,67, "--en--","MD_PORING",1, .@event$;
	monster 'map_begi$,102,67, "--en--","MD_PORING",1, .@event$;
	monster 'map_begi$,101,68, "--en--","MD_PORING",1, .@event$;
	end;
OnMobDead:
	if (mobcount( 'map_begi$, instance_npcname("poring_village_1") + "::OnMobDead" ) > 3)
		end;
	killmonster 'map_begi$, instance_npcname("poring_village_1") + "::OnMobDead";
	mapannounce 'map_begi$, "Emily: Wow~ You are quite strong! You defeated all these Porings!", bc_map, 0xFFFF00;
	sleep 2000;
	mapannounce 'map_begi$, "Emily: But the Poring looks unique somehow. Be careful!", bc_map, 0xFFFF00;
	sleep 2000;

> 	mapannounce 'map_begi$, "Emily: Ah, may be force with you if you go near the blue light column.", bc_map, 0xFFFF00;

Reviewer: ```suggestion
	mapannounce 'map_begi$, "Emily: Ah, there is a unique force if you go near the column of blue light.", bc_map, 0xFFFF00;
```

	monster 'map_begi$,132,185,"--en--","MD_MARIN",1, .@event$;
	monster 'map_begi$,133,185,"--en--","MD_MARIN",1, .@event$;
	monster 'map_begi$,134,185,"--en--","MD_MARIN",1, .@event$;
	monster 'map_begi$,135,185,"--en--","MD_MARIN",1, .@event$;
	monster 'map_begi$,136,185,"--en--","MD_MARIN",1, .@event$;
	monster 'map_begi$,137,185,"--en--","MD_MARIN",1, .@event$;
	monster 'map_begi$,141,185,"--en--","MD_MARIN",1, .@event$;
	monster 'map_begi$,143,185,"--en--","MD_MARIN",1, .@event$;
	monster 'map_begi$,144,185,"--en--","MD_MARIN",1, .@event$;
	monster 'map_begi$,144,185,"--en--","MD_MARIN",1, .@event$;
	monster 'map_begi$,146,185,"--en--","MD_MARIN",1, .@event$;
	monster 'map_begi$,150,185,"--en--","MD_MARIN",1, .@event$;
	monster 'map_begi$,150,185,"--en--","MD_MARIN",1, .@event$;
	end;
OnMobDead:
	if (mobcount( 'map_begi$, instance_npcname("poring_village_3") + "::OnMobDead" ) > 4)
		end;
	killmonster 'map_begi$, instance_npcname("poring_village_3") + "::OnMobDead";
	mapannounce 'map_begi$, "King Poring: Excuse me... But... Would you please get back now?...", bc_map, 0xFFFF;
	sleep 2000;
	mapannounce 'map_begi$, "Emily: What? Who are you?", bc_map, 0xFFFF00;
	sleep 2000;

> 	mapannounce 'map_begi$, "King Poring: I'm a boss of this Poring Village. Sorry for the late introduction.", bc_map, 0xFFFF;

Reviewer: ```suggestion
	mapannounce 'map_begi$, "King Poring: I am the leader of this Poring Village. Sorry for the late introduction.", bc_map, 0xFFFF;
```

	sleep 2000;

> 	mapannounce 'map_begi$, "King Poring: Maybe there is some misunderstand on our print flyers. This Village is so poor and has nothing valuable.", bc_map, 0xFFFF;

Reviewer: ```suggestion
	mapannounce 'map_begi$, "King Poring: Maybe there is some misunderstanding on our flyers. This village is so poor and has nothing valuable.", bc_map, 0xFFFF;
```

	sleep 2000;
	mapannounce 'map_begi$, "Emily: It's for me to judge after looking around more....", bc_map, 0xFFFF00;
	sleep 2000;

> 	mapannounce 'map_begi$, "King Poring: Ugh.... I don't like fight but there is no choice.", bc_map, 0xFFFF;

Reviewer: ```suggestion
	mapannounce 'map_begi$, "King Poring: Ugh.... I don't like fighting but there is no choice.", bc_map, 0xFFFF;
```

		setunitdata $@mobid[.@i], UMOB_SPEED, 200;
	}
	monster 'map_begi$,182,194,"--en--","MD_KING_PORING",1, instance_npcname("poring_village_3") + "::OnBossDead";
	end;
OnBossDead:
	if (mobcount( 'map_begi$, instance_npcname("poring_village_3") + "::OnBossDead" ) > 0)
		end;
	killmonster 'map_begi$, instance_npcname("poring_village_3") + "::OnBossDead";
	enablenpc instance_npcname("#gift");
	end;
}

1@begi,117,108,5	script	Blue Light Column#1	4_ENERGY_BLUE,4,4,{
	end;
OnTouch:
	switch( rand(3) ) {
	case 0:
		unittalk getcharid(3), "" + strcharinfo(0) + " : Ohhhhh!!! Release the parrot inside!!!!";
		break;
	case 1:
		unittalk getcharid(3), "" + strcharinfo(0) + " : I think it's ok to peck someone right now!!";
		break;
	case 2:
		unittalk getcharid(3), "" + strcharinfo(0) + " : Have you been bitten by a mad parrot?";
		break;
	}
	transform 1629,60000;
	specialeffect2 EF_ENHANCE;
	sc_start SC_GLASTHEIM_STATE,30000,20;
	end;
}
1@begi,37,165,5	duplicate(Blue Light Column#1)	Blue Light Column#2	4_ENERGY_BLUE,4,4
1@begi,175,199,5	duplicate(Blue Light Column#1)	Blue Light Column#3	4_ENERGY_BLUE,4,4

1@begi,199,186,5	script	#gift	4_GEFFEN_03,{
	mes "[Emily]";

> 	mes "You can have all item if you find any item in this dungeon. I already have some.";

Reviewer: ```suggestion
	mes "You can have all of the items you find in this dungeon. I already have some.";
```

	next;
	if (isbegin_quest(12418) == 1) {
		mes "[Emily]";

> 		mes "Ah, I found same items now. Let me give you one of them. Just to celebrate your first visit!";

Reviewer: ```suggestion
		mes "Ah, I found some items. Let me give you one of them. Just to celebrate your first visit!";
```

		next;
		if (rand(100) < 50)
			getitem 19238,1;	// PoringTownOnion
		else
			getitem 19239,1;	// PoringTownCarrot
		completequest 12418;
	}
	mes "[Emily]";

> 	mes "I don't need this novice box. You can take it. Then, let's go out side!";

Reviewer: ```suggestion
	mes "I don't need this novice box. You can take it. Now, let's go out side!";
```
